test/partiql-tests-runner/src/main/java/org/partiql/runner/PartiQLTestsRunner.java

```diff
@@ -13,15 +13,6 @@
 )
 public class PartiQLTestsRunner implements Runnable {
 
-    @CommandLine.Option(
-            required = false,
-            description = {"Title of comparison report. Example: 'CROSS-COMMIT', 'CROSS-ENGINE', etc"},
-            paramLabel = "<title>",
-            defaultValue = "CONFORMANCE REPORT",
-            names = {"-t", "--title"}
-    )
-    private String title;
```

Comment:
This still seems like a valuable option for the test runner. Are we able to keep it? The other options don't seem as relevant.